enatega-multivendor-app/src/screens/About/About.js

```
@@ -48,79 +48,39 @@ function About(props) {
-          svgNameL='leftArrow'
-          restaurantImage={restaurantObject.restaurantImage}
-          restaurantName={restaurantObject.restaurantName}
-          deliveryTime={restaurantObject.deliveryTime}
-          total={restaurantObject.total}
-          rating={
-            restaurantObject?.reviews&&restaurantObject?.reviews?.length === 0
-              ? 0
-              : restaurantObject.reviews && restaurantObject?.reviews[0]?.rating
-          }
-        />
+        <ImageHeader iconColor={currentTheme.newIconColor} svgNameL='leftArrow' restaurantImage={restaurantObject.restaurantImage} restaurantName={restaurantObject.restaurantName} deliveryTime={restaurantObject.deliveryTime} total={restaurantObject.total} rating={restaurantObject?.reviews && restaurantObject?.reviews?.length === 0 ? 0 : restaurantObject.reviews && restaurantObject?.reviews[0]?.rating} />
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)